com.github.ytdl_gui.youtubedl_gui.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.ytdl_gui.youtubedl_gui
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: youtubedl-gui
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+modules:
+  - name: x264
+    config-opts: 
+      - --disable-cli
+      - --enable-shared
+    cleanup:
+      - /include
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: b86ae3c66f51ac9eab5ab7ad09a9d62e67961b8a
+
+  - name: ffmpeg
+    config-opts:
+      -  --enable-gpl
+      -  --enable-nonfree
+      -  --disable-static
+      -  --enable-shared     
+      -  --disable-debug
+      -  --disable-doc   
+      -  --enable-libfdk-aac 
+      -  --enable-libfreetype
+      -  --enable-libmp3lame 
+      -  --enable-libopus    
+      -  --enable-libtheora  
+      -  --enable-libvorbis  
+      -  --disable-programs
+      -  --disable-devices
+    sources:
+        - type: archive
+          url: https://ffmpeg.org/releases/ffmpeg-4.4.tar.xz
+          sha256: 06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909
+
+  - python3-youtube-dl.json
+
+  - name: youtubedl-gui
+    buildsystem: cmake
+    sources:
+        - type: archive
+          url: https://github.com/JaGoLi/ytdl-gui/archive/refs/tags/2.5.tar.gz
```

Comment:
Consider adding section for https://github.com/flathub/flatpak-external-data-checker/ here

Reply:
I don't quite understand how to do this